tools/gitlog2changelog.py.in

```diff
@@ -3,16 +3,17 @@
 # Minor changes for NUT by Charles Lepple
 # Distributed under the terms of the GNU General Public License v2 or later
 
-import string, re, os
```

Comment:
Was `string` not needed?

Reply:
It seems so. No functions from `string` are used.

Moreover I have tested generated the ChangeLog for Nut and for Modules (my project) before and after these changes: I obtain the exact same file.